src/main/java/core/mate/academy/model/BulldozerProducer.java

```diff
@@ -0,0 +1,17 @@
+package core.mate.academy.model;
+
+import core.mate.academy.service.MachineProducer;
+import java.util.List;
+
+public class BulldozerProducer implements MachineProducer {
```

Comment:
```suggestion
public class BulldozerProducer implements MachineProducer<Bulldozer> {
```
it's better not to use raw parameters here. This way you will be able to return only Bulldozers from get method